src/patternfly/components/Sequence/sequence.scss

```diff
@@ -80,7 +80,7 @@
   --pf-c-sequence__link--before--m-current--BorderColor: var(--pf-global--primary-color--100);
 
   // Content
-  --pf-c-sequence__content--MinHeight: var(--pf-global--spacer--xl);
+  --pf-c-sequence__content--MinHeight: var(--pf-global--spacer--2xl);
```

Comment:
Do we want a spacer here to set the min height or just a value? If that global value changed would the break this?

Reply:
You're right it would, ill use a value for now